src/explore-education-statistics-admin/src/pages/release/edit-release/data/ReleaseFileUploadsSection.tsx

```diff
@@ -155,40 +159,66 @@ const ReleaseFileUploadsSection = ({
               </>
             )}
 
-            {files &&
-              files.map(file => (
-                <SummaryList key={file.filename}>
-                  <SummaryListItem term="Name">
-                    <h4 className="govuk-heading-m">{file.title}</h4>
-                  </SummaryListItem>
-                  <SummaryListItem term="File">
-                    <ButtonText
-                      onClick={() =>
-                        service
-                          .downloadAncillaryFile(releaseId, file.filename)
-                          .catch(handleApiErrors)
+            <Accordion id="uploaded-files">
+              {files &&
+                files.map((file, index) => (
+                  <AccordionSection
+                    /* eslint-disable-next-line react/no-array-index-key */
+                    key={`${file.title}-${index}`}
+                    headingId={`${file.title}-${index}`}
+                    heading={`${file.title}`}
+                    onToggle={() => {
+                      const accId = `${file.title}-${index}`;
```

Comment:
Move the `accId` up into the above lambda so you can re-use it for `key`, `headingId` and `open`.